src/python/CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ if(USE_CUDA AND CMAKE_CUDA_COMPILER)
   target_link_libraries(python PRIVATE cublasLt cublas cudnn curand cufft cudart)
 endif()
 
+# Avoid warning of Calling FetchContent_Populate(Lib) is deprecated temporarily
```

Comment:
what is the new function to replace it?

Reply:
fetchcontent_makeavailable

Reply:
https://cmake.org/cmake/help/latest/policy/CMP0169.html has more details.